pipeline/data_page/get_all_data_availability.py

```diff
@@ -486,6 +553,12 @@ def main(
     )
     assert omics_summary.index.is_unique
 
+    # Long Reads
+    long_reads_summary = get_long_reads_summary(
+        gcloud_storage_client=gcloud_storage_client,
+        depmap_long_reads_gcloud_loc=depmap_long_reads_gcloud_loc,
+    )
```

Comment:
It looks like depmap_long_reads_gcloud_loc can be an empty list. Is that because long reads data isn't public yet?